src/renderer/api/endpoints/pods.api.ts

```diff
@@ -149,7 +149,16 @@ export interface IPodContainerStatus {
       reason: string;
     };
   };
-  lastState: {};
+  lastState: {
+    [index: string]: object;
```

Comment:
so this is covering `running` and `waiting` states?

Reply:
@nevalla - No, the lastState is only going to show terminated. Do we need the lastState to also show waiting as it can't be running as if a pod terminates and restarts it would be status of running and lastState of terminated.